### PR TITLE
Merging in BridgeServerLogic

### PR DESCRIPTION
**Pull request by alxdarksage**
I believe the exclusions have been moved to the right package given the comment in the original pom.xml file.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -114,6 +84,149 @@
             <version>4.8.0</version>
             <scope>runtime</scope>
         </dependency>
```
Comment (DwayneJengSage) [inline pom.xml]: Minor: Can you move these 3 dependencies with the other runtime dependencies to keep everything in the same place?

File: pom.xml [hunk 2]
```diff
@@ -161,9 +303,22 @@
...
     </repositories>
 
     <build>
+        <extensions>
+            <extension>
+                <groupId>org.springframework.build</groupId>
+                <artifactId>aws-maven</artifactId>
+                <version>5.0.0.RELEASE</version>
+            </extension>
+        </extensions>
```
Comment (DwayneJengSage) [inline pom.xml]: I believe this is used to publish to a Maven repo, which is not necessary.

**Review by DwayneJengSage — APPROVED** [review]
Minor pom issues. Otherwise, looks good.

File: pom.xml (hunk 1 above)
Comment (alxdarksage) [reply pom.xml]: Moved

File: pom.xml (hunk 2 above)
Comment (alxdarksage) [reply pom.xml]: removed

**Pull request merged**